doc/PerformanceTesting.md

```diff
@@ -0,0 +1,169 @@
+# Performance Testing
+
+## Writing a test 
+
+Adding tests , at the moment relies in making modifications in a couple of repositories. 
+
+## Azure-SDK-For-Cpp repo
+
+E.G. https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/storage/azure-storage-blobs 
+
+### Location
+
+Performance tests are located under **"test/perf"** folder under the service folder(e.g. alongside ut folder).
+
+The perf test folder should be added in the parent CMakeLists.txt guarded by the **BUILD_PERF_TEST** flag.
```

Comment:
Is it `BUILD_PERF_TEST` or `BUILD_PERFORMANCE_TESTS`? The text and sample appear to disagree.

Reply:
the second one